packages/mock/src/faker/getters/scalar.ts

```diff
@@ -311,7 +320,12 @@ export const getMockScalar = ({
         mockOptions,
         operationId,
         tags,
-        combine,
+        combine: combine
+          ? {
+              separator: combine.separator,
+              includedProperties: [],
```

Comment:
The problem was that the linked issue used a anyOf schema where both types has the properties `payment_method_id` and `type`. We then saved the includedProperties when traversing down into nested objects, which resulted in the missing properties. I think this is correct, as we don't reset the array, but we still want to be able to add properties of the nested objects.